monitor.py

```diff
@@ -374,17 +374,30 @@ def get_node_status(spy_link):
     
     node_log_url = PROW_VIEW_URL + spy_link[8:] + \
         "/artifacts/" + job_type +"/artifacts/oc_cmds/nodes"
+    
+    pattern=r"(\d+\.\d+)"
+    match=re.search(pattern,spy_link[8:])
+    version=float(match.group(1))
+    if "upgrade" in spy_link:
+        version=version-0.01
+    
  
     try:
         node_log_response = requests.get(node_log_url, verify=False, timeout=15)
         if "NAME" in node_log_response.text:
+            if version > 4.15 and job_platform == "libvirt":
+                no_of_worker="compute-"
```

Comment:
```suggestion
                workers="compute-"
```
Change in all places